pkg/restore/pod_action_test.go

```diff
@@ -26,6 +26,8 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
```

Comment:
Could we use the `builder` package in this file?

I know there's a `Container` builder for init containers. `RestoreHooks` will need an implementation.

Someday I'd like to have the builders generated, because they're kind of a pain to manually make.

Reply:
Used the builder for containers and pods. Leaving the RestoreHooks bit for later.